forms/buildingPermit.json

```diff
@@ -0,0 +1,874 @@
+            {
+              "en": "Select a province",
+              "fr": "Sélectionner une province"
+            },
+            {
+              "en": "Alberta",
+              "fr": "Alberta"
+            },
+            {
+              "en": "British Columbia",
+              "fr": "Colombie-Britannique"
+            },
+            {
+              "en": "Manitoba",
+              "fr": "Manitoba"
+            },
+            {
+              "en": "New Brunswick",
+              "fr": "Nouveau-Brunswick"
+            },
+            {
+              "en": "Newfoundland and Labrador",
+              "fr": "Terre-Neuve-et-Labrador"
+            },
+            {
+              "en": "Northwest Territories",
+              "fr": "Territoires du Nord-Ouest"
+            },
+            {
+              "en": "Nova Scotia",
+              "fr": "Nouvelle-Écosse"
+            },
+            {
+              "en": "Nunavut",
+              "fr": "Nunavut"
+            },
+            {
+              "en": "Ontario",
+              "fr": "Ontario"
+            },
+            {
+              "en": "Prince Edward Island",
+              "fr": "Île-du-Prince-Édouard"
+            },
+            {
+              "en": "Quebec",
+              "fr": "Québec"
+            },
+            {
+              "en": "Saskatchewan",
+              "fr": "Saskatchewan"
+            },
+            {
+              "en": "Yukon",
+              "fr": "Yukon"
+            }
+          ],
+          "required": false
+        }
+      },
+      {
+        "id": 37,
+        "type": "textField",
+        "properties": {
+          "titleEn": "Postal Code",
+          "titleFr": "TBD",
+          "placeholderEn": "",
+          "placeholderFr": "",
+          "description": "",
+          "required": false
+        }
+      },
+      {
+        "id": 38,
+        "type": "textField",
+        "properties": {
+          "titleEn": "Phone number",
+          "titleFr": "TBD",
+          "placeholderEn": "",
+          "placeholderFr": "",
+          "description": "",
+          "required": false
+        }
+      },
+      {
+        "id": 39,
+        "type": "heading",
+        "properties": {
+          "titleEn": "Confirmation",
+          "titleFr": "TBD",
+          "headingLevel": "h2",
+          "isSectional": true,
+          "required": false
+        }
+      },
+      {
+        "id": 40,
+        "type": "checkbox",
+        "properties": {
+          "titleEn": "",
+          "titleFr": "TBD",
+          "descriptionEn": "",
+          "descriptionFr": "",
+          "required": true,
+          "choices": [
+            {
+              "en": "I/We hereby make application for a Building Permit under the Provisions of ÉNational Park Building Regulations, Town of Jasper Land Use Regulations, Municipality of Jasper Bylaws, Fire Regulations, National Park Management Plans, Local Guidelines, and Canadian Environmental Assessment Act that apply.",
```

Comment:
E accent aigu magiscule right before `National Park Building Regulations` on the english side
Otherwise looks good! Thanks Mario